src/main/java/org/jabref/logic/importer/fetcher/DoiFetcher.java

```diff
@@ -140,6 +140,9 @@ public Optional<BibEntry> performSearchById(String identifier) throws FetcherExc
             URLDownload download = getUrlDownload(doiURL);
             download.addHeader("Accept", MediaTypes.APPLICATION_BIBTEX);
 
+            download.addHeader("User-Agent", "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36");
```

Comment:
You could update the header  at and use that https://github.com/JabRef/jabref/blob/861a6ce7062e70a6d16e3530e3f9b2523cf549ff/src/main/java/org/jabref/logic/net/URLDownload.java#L70